make_figures.py

```diff
@@ -179,6 +181,24 @@ def make_timeplot(df_measure, df_prediction):
         legend=dict(x=.05, y=.8, font_size=LABEL_FONT_SIZE,
                     )
     )
+    # vartical line to seprate the last day of measurements from prediction
+    fig.add_shape(
+        # Line Vertical
+        dict(
+            type='line',
+            xref='x',
+            yref='paper',
+            x0=last_day,
+            y0=0,
```

Comment:
maybe aesthetically it would be lighter to start a bit above 0, like `y0=0.1` and `y1=0.9`.